docusaurus/docs/introduction/plugin-lifecycle.md

```diff
@@ -0,0 +1,89 @@
+---
+id: plugin-lifecycle
+title: Life cycle of a plugin
+description: Learn about the life cycle of a Grafana plugin.
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - lifecycle
+  - life cycle
+  - loading
+  - unloading
+  - installation
+sidebar_position: 2.5
+---
+
+# Life cycle of a plugin
+
+This document describes the various phases of a plugin such as installation and loading. We will describe the differences in a plugin's life cycle depending on its type and whether or not it has a [backend](backend.md).  
```

Comment:
<img width="1273" alt="Screenshot 2023-10-26 at 20 16 01" src="https://github.com/grafana/plugin-tools/assets/19860021/b0cde578-964f-4fb0-8b9a-0db90ce3477d">

Seeing this a lot on the backend link... maybe fine, would advise pushing the PR to dev using https://github.com/grafana/plugin-tools/actions/workflows/deploy-to-developer-portal-dev.yml to verify before merging.

Reply:
compilation working for me now, after jackw's PR merged.